ddtrace/internal/_encoding.pyx

```diff
@@ -569,8 +569,9 @@ cdef class MsgpackEncoderV03(MsgpackEncoderBase):
         has_span_type = <bint> (span.span_type is not None)
         has_meta = <bint> (len(span._meta) > 0 or dd_origin is not NULL)
         has_metrics = <bint> (len(span._metrics) > 0)
+        has_parent_id = <bint> (span.parent_id is not None)
 
-        L = 8 + has_span_type + has_meta + has_metrics + has_error
+        L = 7 + has_span_type + has_meta + has_metrics + has_error + has_parent_id
```

Comment:
We reduced this number from 8 to 7 because we now have `has_parent_id` with a conditional of not being 0. If it isn't 0 we'll encode it and need to specify that it's going to be included in the number of values to be encoded.